### PR TITLE
Log Slack Errors

### PR DESCRIPTION
**Pull request by atoff**
Logs the SlackUserAdmin response when invite failed to allow debugging root cause.

### CODE REVIEW

File: app/Http/Controllers/Slack/Registration.php [hunk 1]
```diff
@@ -6,6 +6,7 @@
 use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Facades\Redirect;
 use Illuminate\Support\Facades\Response;
+use Log;
```
Comment (AxonC) [inline app/Http/Controllers/Slack/Registration.php]: Might want to import the full log facade

File: app/Http/Controllers/Slack/Registration.php [hunk 2]
```diff
@@ -35,6 +36,7 @@ public function getNew()
 
             if ($result->ok !== true) {
                 DB::rollBack();
+                Log::error("Error inviting " . $this->account->real_name . " (" . $this->account->id . ") to slack. Response: ". json_encode($result));
```
Comment (AxonC) [inline app/Http/Controllers/Slack/Registration.php]:
```suggestion
                Log::error("Error inviting {$this->account->real_name} ({$this->account->id }) to slack. Response: ". json_encode($result));
```

**Comment by atoff** [comment]
@CalumTowers This is ready to go, I can't seem to stop codeclimate squawking

**Pull request merged**